domains/smutex.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Smalorek",
+           "email": "Smalorek@proton.me",
+           "discord": "286178186645995522"
+        },
+    
+        "record": {
+            "CNAME": "smalorek.github.io"
```

Comment:
You should complete your website first

i visited your repo and saw only a file with
```html
<h1>Test</h1>
```